rasa/core/training/interactive.py

```diff
@@ -778,8 +793,7 @@ def _collect_actions(events: List[Dict[Text, Any]]) -> List[Dict[Text, Any]]:
     try:
         previous_examples = load_data(export_nlu_path)
     except Exception as e:
```

Comment:
I feel like in this exception we should just output to the file given, since we've already validated it. imo this flow:
```
? Do you want to stop?  Export & Quit
? Export stories to (if file exists, this will append the stories) data/stories2.md
? Export NLU data to (if file exists, this will merge learned data with previous training examples) data/nlu2.md
? Export domain file to (if file exists, this will be overwritten) domain2.yml
2019-06-28 16:06:48 ERROR    rasa.core.training.interactive  - An exception occurred while trying to load the NLU data.
? Could not load existing NLU data, please specify where to store NLU data learned in this session (this will overwrite any existing file). Unknown data format for file 'data/nlu2.md'. data/nlu_interactive.md
```
is strange (`data/nlu_interactive.md` is the default that auto-populates). I've already told it where to put it, and it says `if file exists` it will append, so if it doesn't it should just create one and write it without asking again. This only happens for NLU export, not the other two

Reply:
makes sense, I agree :+1:

Reply:
@wochinge was this handled? i’m still running into it

Reply:
Thanks for linking this. Didn't find it when searching for it. will handle it.